developers/ide/intellij.md

```diff
@@ -9,28 +9,42 @@ title: IntelliJ
 
 - git, Maven, IntelliJ and Java 17 are installed
 
-## Install openHAB distribution
+## Build the repositories
 
-1. Install the official [openHAB distribution](https://www.openhab.org/download/)
-1. Start the distribution **in debug mode** (use `./start_debug.sh` instead of `./start.sh` in step 4)
+1. Fork and clone the repositories into a parent directory (Reference `<PARENT_DIR>` from now on for this article). Take only the one(s) you will work on:
 
-This article refers to the directory where you installed the distribution as `<DISTRO_DIR>`.
+    - [openhab addons repository](https://www.github.com/openhab/openhab-addons)
+    - [openhab core repository](https://www.github.com/openhab/openhab-core)
+    - [openhab webui repository](https://www.github.com/openhab/openhab-webui)
 
-## Build the addons repository
+Use the command `git clone https://github.com/<yourgitusername>/openhab-<addons|core|webui>` (replace git user name accordingly).
 
-1. Fork and clone the [openhab addons repository](https://www.github.com/openhab/openhab-addons) into a new directory (Reference `<ADDON_DIR>` from now on for this article) with `git clone https://github.com/<yourgitusername>/openhab-addons` (replace git user name accordingly)
+1. Open IntelliJ, select the file/open, and choose the `<PARENT_DIR>`
+1. Open The Module settings (inside `Project Settings`, or use F4). Click on the + button and select `Import module`.
+1. Choose one of the repository directory you just cloned. select the Maven external model as import format.
+   IntelliJ will start importing and indexing. It will take while, wait until finished. The Module window should now be filled with a bunch of projects.
+   ![Import modules](images/ide_setup_intellij_import_module.png)
+1. Repeat step 2 and 3 for all repositories you cloned.
+1. Use Maven to clean & install projects
```

Comment:
Make sure you use the required maven version (3.8.6 is recommended).

Reply:
Done